notebooks/tutorial_RGDR.ipynb

```diff
@@ -78,72 +84,92 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 15,
+   "execution_count": 4,
    "metadata": {},
    "outputs": [],
    "source": [
-    "field_resampled = s2spy.time.resample(cal, field)\n",
-    "target_resampled = s2spy.time.resample(cal, target)"
+    "field_resampled: xr.Dataset = s2spy.time.resample(cal, field)\n",
+    "target_resampled: xr.Dataset = s2spy.time.resample(cal, target)"
    ]
   },
   {
+   "attachments": {},
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### The target timeseries comes from already pre-clustered land surface temperature data."
+    "### The target timeseries comes from already pre-clustered land surface temperature data.\n",
+    "RGDR requires `DataArray`s as input, as otherwise it is ambiguous which variable should be used.\n",
+    "\n",
+    "When initializing RGDR, the target interval(s) and lag have to be specified as well.\n",
+    "The precursor interval(s) will be determined from the target intervals and lag.\n",
```

Comment:
Add a (more explicit) note here that the maximum lag is constrained by the range of the available precursor intervals?

Reply:
Yes, good point. It is now explicitly mentioned.